.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+Thank you so much for your PR! To help us review, fill out the form to the best of your ability. Also, please make use of the development guide at http://matplotlib.org/devdocs/devel/index.html
+
+<!--- Provide a general summary of your changes in the Title above, for example "Raises ValueError on Non-Numeric Input to set_xlim". Please avoid non-descriptive titles such as "Addresses issue #8576"-->
+
+## PR Summary
+<!--- Please provide at least 1-2 sentences describing the pull request in detail. Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## PR Checklist
+- [ ] Code is tested using Py.test on Py2 and Py3
+- [ ] Code is PEP 8 compliant 
+- [ ] New features are documented-and have an example if plot related
+- [ ] Documentation is sphinx and numpydoc compliant
+- [ ] major new feature: added an entry to doc/users/whats_new.rst
```

Comment:
"For major new features", and below "For backwards incompatible changes to the API"